### PR TITLE
CM-383 update data deletion docs page

### PR DESCRIPTION
**Pull request by rinchen**
add OCfB, clarify encrypted disk, add GDPR data subject deletion requests

### CODE REVIEW

File: src/security/data-deletion.md [hunk 1]
```diff
@@ -1,16 +1,25 @@
 # Data deletion
 
+## Data deletion
```
Comment (Crell) [inline src/security/data-deletion.md]:
```suggestion
# Data deletion
```

Page title gets one #.

Comment (Crell) [inline src/security/data-deletion.md]: Why?  This header is unnecessary.

File: src/security/data-deletion.md [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
+
 Data deletion is handled via our backend providers. When a volume is released back to the provider, the provider will perform a wipe on the data utilizing either [NIST 800-88](https://csrc.nist.gov/publications/detail/sp/800-88/rev-1/final) or [DoD 5220.22-M](http://www.dss.mil/documents/odaa/nispom2006-5220.pdf) depending upon the offering. This wipe is done immediately before reuse.
 
-If your project is utilizing encrypted volumes, the encryption key is destroyed when we release the volume back to the provider adding another layer of protection.
+All projects, except those hosted on Orange Cloud for Business, utilize encrypted volumes. The encryption key is destroyed when we release the volume back to the provider adding another layer of protection.
```
Comment (Crell) [inline src/security/data-deletion.md]:
```suggestion
All projects, except those hosted on Orange Cloud for Business, utilize encrypted volumes. The encryption key is destroyed when we release the volume back to the provider, adding another layer of protection.
```

File: src/security/data-deletion.md (hunk 1 above)
Comment (chadwcarlson) [inline src/security/data-deletion.md]:
```suggestion

```

File: src/security/data-deletion.md [hunk 3]
```diff
@@ -1,16 +1,24 @@
 # Data deletion
 
+
```
Comment (Crell) [inline src/security/data-deletion.md]:
```suggestion
```

**Pull request merged**